home/tests/test_content_import_export.py

```diff
@@ -1694,6 +1694,18 @@ def test_list_item_type_error_message(self, csv_impexp: ImportExport) -> None:
         assert e.value.row_num == 3
         assert e.value.message == ["List item with invalid type 'new_type'"]
 
+    def test_media_link_warning(self, csv_impexp: ImportExport) -> None:
+        """
+        Import a page with media link it should return a warning
+        no error should be raised
+        """
+
+        csv_impexp.import_file("contentpage_media_link_warning.csv")
+
+        page = Page.objects.all()
+
+        assert len(page) > 0
+
```

Comment:
We need to test that we actually get the warning -- perhaps that's something that belongs in the view tests?

Reply:
I need help with this test please